docs/rabbitmq.config.example

```diff
@@ -246,6 +246,10 @@
    %%
    %% {memory_monitor_interval, 2500},
 
+   %% The total memory available can be calculated from the OS resources
+   %% - default option - or provided as a configuration parameter:
+   %% {total_memory_available_override_value, "5000MB"},
```

Comment:
Oh I see. What do you think about changing the comment text to `5000MiB` or `5GiB` so it's clear to the user that those abbreviations are supported?